### PR TITLE
Created the route to get individual chores

### PR DESCRIPTION
**Pull request by Tosvng**

### CODE REVIEW

**Review by Ahsen19 — APPROVED** [review]

File: server/routes/chore.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
 // MIDDLEWARE
 import { verifyToken } from "../middleware/auth.js";
-// CONTROLLER FUNCTIONS
-import { createChore, getAllChores } from "../controller/chore.js";
 
 const router = express.Router();
 
 //ROUTES
-// router.get("/id", ); //get a single chore
+router.get("/id/:id", verifyToken, getSingleChore); //get a single chore
```
Comment (JoshuaDueck) [inline server/routes/chore.js:15]: Does this need to be parameterized to `/:id`?

Comment (Tosvng) [reply server/routes/chore.js:15]: It is, `/id/:id`

Comment (JoshuaDueck) [reply server/routes/chore.js:15]: Oh whoops, I mean like don't have the first ID there. Just have the path be `/api/chores/:id`.

Comment (Tosvng) [reply server/routes/chore.js:15]: I'll make the change

**Review by JoshuaDueck — APPROVED** [review]

**Pull request merged**